### DISCUSSION TITLE
OpenAI Credit Account

### DISCUSSION THREAD

**Discussion by emirhanai** [microsoft/generative-ai-for-beginners#290] [General]
Hello everyone,
I do not use my OpenAI account, which contains 2510 USD credit, because it is redundant. For this reason, I want to sell urgently. If anyone wants to buy it, I can deliver the account for 1500 USD.

Thanks.

ETSY link: [ETSY Market Listing](https://www.etsy.com/listing/1647873218/openai-2510usd-credits-accounts)
Contact (e-mail): [emirhan@piya.ai](mailto:emirhan@piya.ai)

Below is the credit image:

![image](https://github.com/Significant-Gravitas/AutoGPT/assets/72056400/776e8013-c056-4bf3-b6cd-5af4d5f0013b)

**Comment by Habte36** [comment]
How for my purpose do my website
